symbols/overlay29.yml

```diff
@@ -486,9 +503,18 @@ overlay29:
         NA: 0x22EBD08
         JP: 0x22ED370
       description: |-
-        The main function which executes the actions that take place in a fractional turn. Called in a loop by InitializeDungeon while FloorIsOver returns false.
+        The main function which executes the actions that take place in a fractional turn. Called in a loop by InitializeDungeon while IsFloorOver returns false.
```

Comment:
`InitializeDungeon` -> `RunDungeon` to be consistent with the name change

Reply:
Fixed, thanks.